channeldb/invoices.go

```diff
@@ -824,7 +848,7 @@ func (d *DB) QueryInvoices(q InvoiceQuery) (InvoiceSlice, error) {
 // The update is performed inside the same database transaction that fetches the
 // invoice and is therefore atomic. The fields to update are controlled by the
 // supplied callback.
-func (d *DB) UpdateInvoice(paymentHash lntypes.Hash,
+func (d *DB) UpdateInvoice(paymentHash lntypes.Hash, payAddr *[32]byte,
```

Comment:
As a preparation for later adding `setId`, maybe the total code delta will be smaller if an `InvoiceRef` struct is introduced containing the two (and later three) fields?